server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -0,0 +1,736 @@
+
+.Take site offline
+
+There are 2 ways to take the site offline.
+
+**Manually by admin** - Admin can use the `jconsole` or other tool and run some JMX operations to manually take the particular site offline.
+This is useful especially if the outage is planned. With `jconsole` or CLI, you can connect to the `jdg1` server and take the `site2` offline. 
+More details about this are available in the link:https://access.redhat.com/documentation/en-us/red_hat_jboss_data_grid/7.1/html/administration_and_configuration_guide/set_up_cross_datacenter_replication#taking_a_site_offline[JDG documentation]. 
+ 
+WARNING: This has turned off the backup to `site2` for the cache `sessions`. The same steps usually need to be done for all the other {project_name} caches mentioned [here](#sync-or-async-backups).
+
+**Automatically** - After some amount of failed backups, the `site2` will usually be taken offline automatically. This is done due the configuration of `take-offline` element inside the cache configuration as configured in link:jdgsetup[JDG server setup].
+
+```
+<take-offline min-wait="60000" after-failures="3" />
+```
+
+This example shows that the site will be taken offline automatically for the particular single cache if there are at least 3 subsequent failed backups and there is no any successful backup within 60 seconds.
+
+Automatically taking a site offline is useful especially if the broken network between sites is unplanned. The disadvantage is that there will be some failed backups until the network outage is detected, which could also mean  failures on the application side. 
+For example, there will be failed logins for some users or big login timeouts. Especially if `failure-policy` with value `FAIL` is used.
+
+WARNING: The tracking of whether a site is offline is tracked separately for every cache.
+ 
+
+.Take site online
+
+Once your network is back and `site1` and `site2` can talk to each other, you may need to put the site online. This needs to be done manually through JMX or CLI in similar way as taking a site offline.
+Again, you may need to check all the caches and bring them online.
+
+Once the sites are put online, it's usually good to:
+* Do the link:statetransfer[state transfer].
+* Manually link:clearcache[clear the Keycloak caches]. 
+
+
+[[statetransfer]]
+==== State transfer
+
+State transfer is a manually required step. JDG does not do this automatically, for example during split-brain, it is only the admin who may decide which site has preference and hence if state transfer needs to be done bidirectionally between both sites or just unidirectionally, as in only from `site1` to `site2`, but not from `site2` to `site1`. 
```

Comment:
"manually required step" - should it be "required manual step"?
JDG -> {jdgserver_name}